modules/olmv1-installing-an-operator.adoc

```diff
@@ -20,25 +20,29 @@ You can install an extension from a catalog by creating a custom resource (CR) a
 .Example `pipelines-operator.yaml` CR
 [source,yaml]
 ----
-apiVersion: olm.operatorframework.io/v1alpha1
+apiVersion: olm.operatorframework.io/v1
```

Comment:
@michaelryanpeter when I review https://88848--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/ce/managing-ce.html, I find 
```yaml
opm render <catalog_registry_url>:<tag> | jq -s '.[] | select( .schema == "olm.package")
```
is not correct. it miss "'"

the correct is 
```yaml
opm render <catalog_registry_url>:<tag> | jq -s '.[] | select( .schema == "olm.package")'
```

I know it has nothing with this PR, but I do not want to create the bug which is late to fix, so I raise it here.

thanks to understand it.

Reply:
I agree. Let's fix it while we're here!